src/config/steps/fing.coffee

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  name: 'fing',
-  view : 'steps/fing'
-  isActive: (instance) ->
-    instance.attributes \
-      && instance.attributes.context \
-      && instance.attributes.context.toLowerCase() is 'fing'
+    name: 'fing',
+    view : 'steps/fing'
+    isActive: (instance) ->
+        instance.attributes \
```

Comment:
I do not understand that way of indenting code, to me it leads to poorly indented code like : 
```
if (
  toto
    && tata
) {
// do stuff
}
```